2-asp-net-core-api-user-signin/README.md

```diff
@@ -94,33 +111,43 @@ If you want to modify the payloads `issuance_request_config.json` and `presentat
 
 Make sure you copy the `ClientId`, `ClientSecret` and `TenantTd` you copied when creating the app registration to the `appsettings.json` as well.
 
+Make sure you copy the `ClientId` and `TenantTd` you copied when creating the app registration to the `appsettings.json` **AzureAd** section as well.
+
 ## Running the sample
 
-1. Open a command prompt and run the following command:
+1. Open a command prompt, run ngrok to set up a URL on 3000. You can install ngrok globally by using the [ngrok npm package](https://www.npmjs.com/package/ngrok/).
 ```Powershell
-dotnet build "AspNetCoreVerifiableCredentials.csproj" -c Debug -o .\bin\Debug\net5
-dotnet run
+ngrok http 3000
```

Comment:
Alternatively if we want to improve the directions even for future samples where multiple registrations are needed, we can recommend using paid permanent ngrok URL. Something like below.. 

==
Open a command prompt, run ngrok to expose your local web server running on port 3000 via a public URL. For example: 
    * Win: `./ngrok http 3000 -host-header=localhost:3000 -subdomain="contoso"`
    * Mac: `/ngrok http 3000 -host-header=localhost:3000 -subdomain="contoso"
    
`NOTE: The free ngrok plan will generate a new URL every time you run it, which requires you to follow Step 2 below every time you run the sample. A paid account with a permanent ngrok URL is recommended.
==